scripts/Diffraction/isis_powder/abstract_inst.py

```diff
@@ -67,7 +67,9 @@ def _focus(self,
                run_number_string,
                do_van_normalisation,
                do_absorb_corrections,
-               sample_details=None):
+               sample_details=None,
+               absorb_method="Mayers",
```

Comment:
The HRPD instrument uses an algorithm called HRPDSlabCanAbsorption to calculate its absorption corrections so the Mayers method isn't perhaps used widely enough to have it as a default here. I think everything should work OK if you just use `None` as the default value here.
Also this has got me thinking - perhaps the absorb_method should actually be called empty_container_subtraction_method with values PaalmanPings and Simple. We have a few absorption correction algorithms using different types of calculation (eg MayersSampleCorrection and MonteCarloAbsorption are both Monte Carlo calculations, HRPDSlabCanAbsorption uses numerical integration and there are others that just uses pre-calculated lookup tables) and in theory they could all be used within a PaalmanPings approach or a simple empty can subtraction approach.